contracts/REXReferral.sol

```diff
@@ -23,6 +23,7 @@ contract REXReferral is Ownable {
     mapping(string => uint256) public affiliateIdToAffiliate;
     mapping(address => uint256) public userToAffiliate;
     mapping(address => bool) public isUserOrganic;
+    mapping(address=>bool) public allowedUsers;
```

Comment:
Can you not use "users" here and use `approvedApps`?

Reply:
Done 👍

Reply:
This is not needed anymore 😄